lib/src/model/locatable.dart

```diff
@@ -6,6 +6,13 @@ import 'package:analyzer/dart/element/element.dart' show Element;
 
 /// Something that can be located for warning purposes.
 mixin Locatable {
+  /// The model element(s) from which we will get documentation.
```

Comment:
Why not link [ModelElement] and the other references in backticks like the original comment?